makefiles/pseudomodules.inc.mk

```diff
@@ -91,3 +91,7 @@ PSEUDOMODULES += prng_%
 
 # STM32 periph pseudomodules
 PSEUDOMODULES += stm32_periph_%
+
+# declare periph submodules as pseudomodules, but exclude periph_common
+PSEUDOMODULES += periph_%
+NO_PSEUDOMODULES += periph_common
```

Comment:
I still have to smile at this construct, as in 'how to implement the word 'except' in Makefiles' - ALL cars are fast, except the ones that are not :-)

I could think of one alternative to this, being to state the peripheral modules explicitly. There are only 10 or so and this number is most likely not to grow significantly... I would think that 10 explicit entries could be more run-time friendly than another filter operation + wildcard handling?!

Reply:
Do you mind a lot? We'd have to list all the exceptions (e.g., hwrng_rng[ab], rtt_lptim) as well. I'd rather use this construct (NO_PSEUDOMODULE) for this one exception (periph_common)...

Reply:
ok, leave it for now. We can always revisit :-)

Reply:
This [this webcomic](https://blog.toggl.com/world-created-programmer/) reminds me a lot of our build system ;-).